app/build.gradle

```diff
@@ -28,11 +30,11 @@ android {
         }
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_17
```

Comment:
do we have to change this?

Reply:
nope, my setup was giving me issues so I had to